tsl/test/expected/continuous_aggs_bgw.out

```diff
@@ -193,7 +193,7 @@ BEGIN
     ELSEIF (num_runs > expected_runs) THEN
         RAISE 'num_runs > expected';
 	ELSE
-		PERFORM pg_sleep(0.1);
+		PERFORM pg_sleep(1.0);
```

Comment:
while we could probably decrease the polling rate, a 10x increase in polling period seem a bit much. If #spins is 100 we're now waiting up to 100 s to time out, which seems too long. Maybe a more moderate increase here makes sense and then also change the amount of spins for a more reasonable timeout?

I am also noting that the old polling period (0.1) is still used elsewhere. Would be good to harmonize this so that we have a standard rate and amount of spins.

Reply:
shouldnt we increase the number of polls instead of increasing the interval, there is also TEST_SPINWAIT_ITERS to set this globally and its part of the function call signature of the test functions

Reply:
I do not understand why we need to poll every 100ms. At worst, we make the test run 900ms longer, which is not a significant amount.

If we are waiting 100 seconds for a timeout, there is something broken with the code. In other words, we will usually get a success earlier and this will not be the common case.

However, changing to use 1.0 seconds everywhere seems sensible.

Reply:
@mkindahl 900ms might not seem like much in isolation. But small changes in timeouts can easily add up if  they happen often. Just saying. 

That's why I think we should aim for tests to pass as quickly as possible without being flaky. We already have long run times, let's not add to that unnecessarily.

Reply:
I'll go ahead and pile on here.  It seems the correct fix here is to increase the number of polls not the polling interval.

Reply:
We cannot force tests to be faster by reducing the timeout, they will take the time they take. Reducing the timeout will just increase the flakiness. A high polling rate risk aggravating the problem and increase the execution time since the check itself pulls in a lot of machinery to just perform the check and hence pushes out other work that could have been done.

However, I have changed the patch to increase the default spin count instead.